api/src/pcapi/alembic/versions/20240110T165515_b429bc5435d6_create_playlists.py

```diff
@@ -0,0 +1,66 @@
+"""Create the playlist table to fetch data from BigQuery
+and allow Adage's discovery page to get the playlists
+directly from the DB.
+"""
```

Comment:
[détail] Mieux vaut une description courte sur une seule ligne, car cette docstring est reprise dans `alembic history` (entre autres), et les sauts de ligne rendent alors la sortie peu lisible. Suggestion, toute simple : 

```suggestion
"""Create `collective_playlist` table"""
```